New package: TestRunner v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: TestRunner
- Repository: https://github.com/aviatesk/TestRunner.jl
- Created by: @aviatesk
- Version: v0.1.0
- Commit: 603ee9bd23c80517980563b6873c1aad79942552
- Reviewed by: @aviatesk
- Reference: https://github.com/aviatesk/TestRunner.jl/commit/603ee9bd23c80517980563b6873c1aad79942552#commitcomment-173100155
- Description: A selective test runner for Julia
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6afc67fa18184fe6e91e8eb0fc90bd15937115db6651340ee014c9b9e83703f020609aa274b41361e7059f251a9731e171e7d75c51a544bca65f8a5211650c5099f2897c4089949a751dc89c1c282aed63076cbd9df3c81cbbac734cd4813b8e5d63745e65af9398c2a154be2b77822fd14eca034f44d5b5010dd8664542f35170590b784c577f4646a958cb9d862c83536b0cd9270069a77ab95293c0041d8a13bb3ad17b7e1ddf8b9055fc8a19259e4 -->

aviatesk commented:
I think we should do all of them ideally, but I won't have time for them in the next week or so. Is it better to do them before this registration is released?

JuliaRegistrator committed:
New package: TestRunner v0.1.0

UUID: 4efbe77e-ddf3-4ce7-be91-16409ca9e85f
Repo: https://github.com/aviatesk/TestRunner.jl.git
Tree: 94113b7dab87ed1bb74bf67d5f9c33f5445712e3

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
